rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -495,18 +491,14 @@ impl Controller {
         let api_url = self.advanced_settings.api_url.clone();
         tracing::info!(api_url = api_url.to_string(), "Starting connlib...");
 
-        let mut connlib = ipc::Client::connect(
+        let connlib = ipc::Client::connect(
             api_url.as_str(),
             token,
             callback_handler.clone(),
             tokio::runtime::Handle::current(),
         )
         .await?;
 
-        connlib
-            .set_dns(client::resolvers::get().unwrap_or_default())
-            .await?;
```

Comment:
We can delete this because we will just do it on the first tick of the timer?

Reply:
Maybe. I don't think I have a test to prove that yet, so I'll leave it. If DNS control is up in the Client now, maybe I can try to enforce in the Client that the "Firezone connected" notification or the systemd ready notification only fires after we definitely control DNS. Maybe in another PR